### PR TITLE
Perl corelibs

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl4-corelibs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "perl4-corelibs" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.003.tar.gz
```
Comment (bgruening) [inline recipes/perl4-corelibs/meta.yaml]: you can use the {{version here}}

File: recipes/perl4-corelibs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  commands:
+    - perl -e 'require "getopts.pl"' 
+
+about:
+  home: http://search.cpan.org/~zefram/Perl4-CoreLibs-0.003/lib/Perl4/CoreLibs.pm
+  description: "This is a collection of .pl files that have historically been bundled with the Perl 
+                core but are planned not to be so distributed with core version 5.15 or later." 
+
+
```
Comment (bgruening) [inline recipes/perl4-corelibs/meta.yaml]: Can you remove the trailing line breaks? Also please include a license information.

File: recipes/perl4-corelibs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - perl-app-cpanminus
+    - perl-module-build
+
+  run:
+    - perl-threaded
+
+
+test:
+  commands:
+    - perl -e 'require "getopts.pl"'
```
Comment (bgruening) [inline recipes/perl4-corelibs/meta.yaml]: Can you not use something like this? https://github.com/bioconda/bioconda-recipes/pull/3362/files#diff-15ac499c35dfd349ebcdf8aa6252f868R41

**Comment by jerowe** [comment]
I tried this one out too, and although I could get it to install, activate the env, and require the libraries in a interactive perl shell, I cannot get the tests to pass in the meta.yaml file.

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**